client/src/components/CreateSession.jsx

```diff
@@ -124,31 +172,60 @@ class CreateSession extends React.Component {
 
             : <div>
                 <span>Session Name: {this.state.sessionName}</span>
-                <form>
-                  <label htmlFor="searchQuery">Search restaurants by name:</label>
-                  <input
-                    id="restaurant-search"
-                    type="text"
-                    inputMode="search"
-                    name="searchQuery"
-                    value={this.state.searchQuery}
-                    onChange={this.handleChange}
-                    data-testid="restaurant-search-input"
-                  />
-                  <input type="submit" value="Search" onClick={this.getRestaurants} />
-                </form>
-
-                {this.state.showSuggested ? 'Suggested Restaurants Nearby:' : 'Search Results:'}
-                <ul>
-                  {this.state.restaurants.map(restaurant =>
-                    <li key={restaurant.restaurant_id}>
-                      <span>{restaurant.restaurant_name} - {restaurant.address.formatted}</span>
-                      <Link to='/add-guests' >
-                        <button onClick={() => this.selectRestaurant(restaurant)}>Select</button>
-                      </Link>
-                    </li>
-                  )}
-                </ul>
+                {!this.state.geolocationProcessComplete && <div>Loading...</div>}
+
+                {this.state.geolocationProcessComplete &&
+                  <div>
+                    <form>
+                      <label htmlFor="searchQuery">Restaurant Name:</label>
+                      <input
+                        id="restaurant-search"
+                        type="text"
+                        inputMode="search"
+                        name="searchQuery"
+                        value={this.state.searchQuery}
+                        onChange={this.handleChange}
+                        data-testid="restaurant-search-input"
+                      />
+
+                      <label htmlFor="zipCode">Zip Code:</label>
+                      {this.props.hostGeo
+                        ? <input
+                            id="zip-code"
```

Comment:
The `id` has to match the label's `htmlFor`: 

> To associate the <label> with an <input> element, you need to give the <input> an id attribute. The <label> then needs a for attribute whose value is the same as the input's id.

<https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label>

Reply:
Another good catch, updated all my input id's now